modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -0,0 +1,32 @@
+= Configure Azure Client Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use dynamic client registration to configure Azure client management with Anypoint Platform.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-client-providers.adoc[]
+
+. Click *Add Client Provider*, and then select *OpenID Connect DCR for Azure*.
++
+The *Add OIDC Azure client provider* page appears.
++
+. Enter a name and description for your client provider.
+. Enter the following values from your identity provider’s configuration:
+.. *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
+.. *Tenant ID for Azure AD*: The tenant ID from Azure Active Directory. For more information on obtaining your tenant ID, see https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant[How to find your Azure Active Directory tenant ID].
+.. *Client ID*: The client ID for an existing client in your IdP capable of introspection of all tokens from all clients. Use a client that has permissions to create applications in Azure.
```

Comment:
this client ID is not for introspection of all tokens from all clients, this client ID is for creating applications in Azure AD.

Reply:
```suggestion
.. *Client ID*: The client ID for an existing client in your IdP capable of creating applications in Azure.
```